### DISCUSSION TITLE
How do I use/configure frameworks for Deploy-Now?

### DISCUSSION THREAD

**Discussion by Lyso95** [ionos-deploy-now/ionos-deploy-now#97] [Q&A] [answered]
Hey, 

am I just too dump to host an application? This "One-Click-Thing" isn´t working for me.

I am continuously running into: 403 Forbidden
Forbidden
You don't have permission to access this resource.

When trying to host something. I managed to publish the template of next:
https://home-5013132557.app-ionos.space/ 

But when I try to do something else like: 
https://github.com/Lyso95/next-portfolio
with
https://nextra.site/

or

https://github.com/Lyso95/reactvite_test
with
https://vitejs.dev/

I get stuck everytime...

Can you give me a hint? Found this in docs, but don´t get the clue how to config.
https://docs.ionos.space/docs/apache-configuration-htaccess/

Thank you for you help!

**Comment by agaengel** [comment, marked as answer]
Hi @Lyso95 
the deployment of https://github.com/Lyso95/next-portfolio but your next project does not create a index.html

you can check the content of the running website with the deployment viewer: https://docs.ionos.space/docs/deployment-viewer/#deployment-viewer

I hope you can fix your build an publish you website with deploy now

Regards
Alex

**Reply by Lyso95** [reply]
Facepalm! Should have used "out" and not "public" as deployment folder... Thank you very much!
